Commands/Search/GetSiteHistoricalVersionsStatus.cs

```diff
@@ -0,0 +1,56 @@
+﻿#if !ONPREMISES
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.SharePoint.Client;
+using Microsoft.SharePoint.Client.Search.Administration;
+using PnP.PowerShell.CmdletHelpAttributes;
+
+namespace PnP.PowerShell.Commands.Search
+{
+    [Cmdlet(VerbsCommon.Get, "PnPSiteHistoricalVersionsStatus", DefaultParameterSetName = "Xml")]
```

Comment:
You don't need the DefaultParameterSetName here, as you only have one set, and no. parameters.

Reply:
Removed it in latest commit